frontend/amundsen_application/static/js/components/ResourceListItem/TableListItem/index.tsx

```diff
@@ -24,9 +24,26 @@ export interface TableListItemProps {
   disabled?: boolean;
 }
 
-export const getLink = (table, logging) =>
-  `/table_detail/${table.cluster}/${table.database}/${table.schema}/${table.name}` +
-  `?index=${logging.index}&source=${logging.source}`;
+export const getName = (table) => {
+  /*
+  this function get's the table name from the key to preserve original
+  capitalization since search needs the names to be lowercase for analysis
+  */
+  const splitKey = table.key.split('/');
+  const keyName = splitKey[splitKey.length - 1];
+  if (keyName.toLowerCase() === table.name) {
+    return keyName;
+  }
+  return table.name;
+};
+
+export const getLink = (table, logging) => {
+  const name = getName(table);
```

Comment:
nit: I would just inline the call instead of creating a temporal.

Reply:
I tried but had a weird lint issue I couldn't fix or auto fix